<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicTab</class>
 <widget class="QWidget" name="BasicTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="main_layout">
   <item>
    <widget class="QWidget" name="left_panel">
     <layout class="QVBoxLayout" name="left_layout">
      <item>
       <widget class="QGroupBox" name="location_group">
        <property name="title">
         <string>Repository Location</string>
        </property>
        <layout class="QVBoxLayout" name="location_layout">
         <item>
          <layout class="QHBoxLayout" name="location_path_layout">
           <item>
            <widget class="QLineEdit" name="repo_location_edit">
             <property name="placeholderText">
              <string>Path where repository will be created</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browse_location_btn">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="folder_name_layout">
           <item row="0" column="0">
            <widget class="QLabel">
             <property name="text">
              <string>Folder Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="folder_name_edit">
             <property name="placeholderText">
              <string>Local folder name (allows spaces)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="repo_group">
        <property name="title">
         <string>Repository Information</string>
        </property>
        <layout class="QFormLayout" name="repo_layout">
         <item row="0" column="0">
          <widget class="QLabel">
           <property name="text">
            <string>Repository Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="repo_name_edit">
           <property name="placeholderText">
            <string>Repository name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="description_edit">
           <property name="placeholderText">
            <string>Repository description (optional)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="visibility_group">
        <property name="title">
         <string>Visibility</string>
        </property>
        <layout class="QVBoxLayout" name="visibility_layout">
         <item>
          <widget class="QRadioButton" name="visibility_public_radio">
           <property name="text">
            <string>Public</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="visibility_private_radio">
           <property name="text">
            <string>Private</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="license_group">
        <property name="title">
         <string>License</string>
        </property>
        <layout class="QVBoxLayout" name="license_layout">
         <item>
          <widget class="QRadioButton" name="license_none_radio">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="license_mit_radio">
           <property name="text">
            <string>MIT License</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="license_apache_radio">
           <property name="text">
            <string>Apache 2.0 License</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="license_gpl_radio">
           <property name="text">
            <string>GPL 3.0 License</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="project_type_group">
        <property name="title">
         <string>Project Type</string>
        </property>
        <layout class="QVBoxLayout" name="project_type_layout">
         <item>
          <widget class="QRadioButton" name="project_type_other_radio">
           <property name="text">
            <string>Other</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="project_type_python_lib_radio">
           <property name="text">
            <string>Python Library</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="project_type_python_cli_radio">
           <property name="text">
            <string>Python CLI Tool</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="project_type_js_radio">
           <property name="text">
            <string>JavaScript/Node.js Package</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="project_type_web_radio">
           <property name="text">
            <string>Web Application</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="project_type_data_radio">
           <property name="text">
            <string>Data Science Project</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="project_type_docs_radio">
           <property name="text">
            <string>Documentation Site</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gitignore_group">
        <property name="title">
         <string>Gitignore Template</string>
        </property>
        <layout class="QVBoxLayout" name="gitignore_layout">
         <item>
          <widget class="QComboBox" name="gitignore_combo"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="topics_group">
        <property name="title">
         <string>Topics</string>
        </property>
        <layout class="QVBoxLayout" name="topics_layout">
         <item>
          <widget class="QLineEdit" name="topics_edit">
           <property name="placeholderText">
            <string>Comma-separated topics (e.g., python, gui, qt)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="right_panel">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="right_layout">
      <item>
       <widget class="QLabel" name="help_label">
        <property name="text">
         <string>ℹ️ Information &amp; Help</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold; font-size: 14px; padding: 10px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="help_browser">
        <property name="styleSheet">
         <string>QTextBrowser {
    background-color: #f8f9fa;
    border: 1px solid #dee2e6;
    border-radius: 6px;
    padding: 15px;
    font-size: 12px;
    line-height: 1.4;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
